types/node-mongodb-fixtures/node-mongodb-fixtures-tests.ts

```diff
@@ -0,0 +1,42 @@
+import * as Fixtures from 'node-mongodb-fixtures';
```

Comment:
Use the commonjs syntax `import Fixtures = require('node-mongodb-fixtures')`. According to the Ecmascript spec, the `import *` syntax gives you an object that isn't callable or constructable, although Typescript emits the correct code when emitting commonjs modules.

Reply:
@sandersn Thanks for the review and explanation! Committed the change, please review again.